app/static/js/grid_scripts.js

```diff
@@ -0,0 +1,48 @@
+const tolerance = 1;
+let last = 0;
+let lastIndex = 0;
+
+function setToggle(elementClass, targetElement, classToAdd) {
+    let allDays = document.getElementsByClassName(elementClass);
+    for (let i = lastIndex; i < allDays.length; ++i) {
+        allDays[i].addEventListener("click", function () {
+            let target = document.getElementById(targetElement);
+            target.classList.toggle(classToAdd);
+        })
+    }
+    lastIndex += allDays.length - lastIndex;
+}
+
+document.addEventListener(
+    'DOMContentLoaded', function () {
+        setToggle("day", "day-view", "day-view-visible");
+    }
+)
+
+function loadWeek(lastDay) {
+    if (lastDay === last) {
+        return false;
+    }
+    let path = '/calendar/month/' + lastDay;
+    last = lastDay;
+    fetch(path).then(function (response) {
+        return response.text();
+    }).then(function (html) {
+        let newDays = document.createElement('html');
```

Comment:
const, outside the fetch (after `path`)

Reply:
I'm not sure I understand what so u mean by "outside the fetch (after path)".